nntrainer/models/model_loader.cpp

```diff
@@ -456,7 +455,38 @@ int ModelLoader::loadFromIni(std::string ini_file, NeuralNetwork &model,
  * @brief     load all of model and dataset from given config file
  */
 int ModelLoader::loadFromConfig(std::string config, NeuralNetwork &model) {
-  return loadFromConfig(config, model, false);
+
+  NNTR_THROW_IF(model_file_context != nullptr, std::runtime_error)
```

Comment:
can you change this throw to `return error` or change other return error to throw for this function to keep it consistent?

Reply:
Done to `return error`